src/defaults.mligo

```diff
@@ -3,29 +3,45 @@
 
 #include "types.mligo"
 
-let default_config : config = {
+let default_config (max_quorum, min_quorum, max_voting_period, min_voting_period : nat * nat * nat * nat): config = {
```

Comment:
see the suggestion before, I'd create here an `initial_config_data` type to carry this info (and use it as part of the `initial_data` type as well perhaps).